### ISSUE TITLE
Error property 'FullName' cannot be found on this object (docker microsoft/bcsandbox:fr)

### ISSUE THREAD

**Issue by xgaronnat** [Microsoft/navcontainerhelper#212]
Hi,

When I create a new docker with the script below, the script failed with this error :
> New-NavContainer : **The property 'FullName' cannot be found on this object**. Verify that the property exists.
> At C:\Users\xgaronnat8119\Downloads\Untitled1.ps1:18 char:1
> + New-NavContainer -accept_eula:$accept_eula `
> + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>     + CategoryInfo          : NotSpecified: (:) [New-NavContainer], PropertyNotFoundException
>     + FullyQualifiedErrorId : PropertyNotFoundStrict,New-NavContainer
> 

The script is issued from CreateBCSandbox.ps1 :
```
Set-ExecutionPolicy Unrestricted
install-module navcontainerhelper -force
$accept_eula = $true
 $containername = 'MyContainer'
$navdockerimage = 'microsoft/bcsandbox:fr'
$appbacpacuri = ''
$tenantbacpacuri = ''
 
$additionalParameters = @()
if ($appbacpacuri -ne '' -and $tenantbacpacuri -ne '') {
    $additionalParameters = @("--env appbacpac=""$appbacpacuri""","--env tenantBacpac=""$tenantbacpacuri""")
}
 
$credential = get-credential -UserName $env:USERNAME -Message "Using Windows Authentication. Please enter your Windows credentials."
New-NavContainer -accept_eula:$accept_eula `
                 -containername $containername `
                 -auth Windows `
                 -Credential $credential `
                 -includeCSide `
                 -alwaysPull `
                 -doNotExportObjectsToText `
                 -usessl:$false `
                 -updateHosts `
                 -assignPremiumPlan `
                 -shortcuts Desktop `
                 -imageName $navdockerimage `
                 -additionalParameters $additionalParameters
 
Setup-NavContainerTestUsers -containerName $containername -password $credential.Password

```
Web client is not working and only 3 icons were created on the desktop (Cmd Prompt, PS Prompt and Web client).

Thanks for your help,
Xavier.

**Comment by freddydk** [comment]
Nothing seems to be wrong with the script - could you try to re-run it?
and maybe try with :us instead of :fr for a test and give me the result.
Thanks

**Comment by xgaronnat** [comment]
Looks weird... after 2 days spent trying create this docker (with really slow pulling time on image), it's now running smoothly. I can't say why, nevertheless it's working now ;-)

**Comment by AbsolutelyFreeWeb** [comment]
Hi, I'm getting the same exact problem when trying to create a new micorosft:bcsandbox:us container

Reading CustomSettings.config from ConceptAG
Creating Desktop Shortcuts for ConceptAG
New-NavContainer : The property 'FullName' cannot be found on this object. Verify that the property exists.
At line:1 char:1
+ New-NavContainer -accept_eula:$accept_eula `
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [New-NavContainer], PropertyNotFoundException
    + FullyQualifiedErrorId : PropertyNotFoundStrict,New-NavContainer

**Comment by AbsolutelyFreeWeb** [comment]
OK it worked for me too now. The log output is different. 
when it worked I got:
Starting NAV Service Tier
Creating DotNetCore Web Server Instance
Enabling Financials User Experience
Creating http download site
Creating Windows user amdev
Setting SA Password and enabling SA
Creating amdev as SQL User and add to sysadmin
Creating SUPER user
Assign Premium plan for AMDEV
Container IP Address: 172.x.x.x

when it wasn't working I got:
Starting NAV Service Tier
Creating DotNetCore Web Server Instance
Enabling Financials User Experience
Creating http download site
Creating Windows user amdev
Setting SA Password and enabling SA
Creating amdev as SQL User and add to sysadmin
Initializing...
Restarting Container
PublicDnsName unchanged
Hostname is XxxxxxAG
PublicDnsName is XxxxxxAG
Using NavUserPassword Authentication
Starting Local SQL Server
Starting Internet Information Server
Starting NAV Service Tier
Container IP Address: 172.x.x.x

Does this difference help you to see what might have gone wrong?
